style/index.html

```diff
@@ -45,5 +47,16 @@
         data-recalc-dims="1"
         style="position: absolute; top: 0; right: 0; border: 0; z-index: 100"
     /></a>
+    <div id="geocoder-search-panel">
+      <form>
+        <input
+          id="geocoder-search-input"
+          type="text"
+          placeholder="Search"
+          autocomplete="off"
+        />
+      </form>
+      <div id="geocoder-live-results" />
```

Comment:
This is incorrect for elements which are not self-closing in HTML5.

```suggestion
      <div id="geocoder-live-results"></div>
```